pages/services/prometheus/0.1.0-2.3.1/supported-versions/index.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: layout.pug
+navigationTitle:  Supported Versions
+title: Supported Versions
+menuWeight: 120
+excerpt: Understanding DC/OS Prometheus Services package versioning
+featureMaturity:
+enterprise: false
+---
+
+# Supported Versions
+
+## Package Versioning Scheme
+
+    Prometheus: 2.3.1
+    DC/OS: 1.10 and 1.11
+
+Packages are versioned with an a.b.c-x.y.z format, where a.b.c is the version of the DC/OS integration and x.y.z indicates the version of Prometheus.
+For example, 0.2.0-2.2.1 indicates version 2.2.1 of the DC/OS integration and version 2.2.1 of Prometheus.
```

Comment:
`For example, 0.2.0-2.2.1 indicates version 2.2.1 of the DC/OS integration and version 2.2.1 of Prometheus` - Versions are incorrect.

Reply:
Updated